src/pass/storage_rewrite.cc

```diff
@@ -799,23 +819,17 @@ class StoragePlanRewriter : public IRMutator {
     if (const_nbits != 0) {
       // constant allocation.
       auto begin = const_free_map_.lower_bound(const_nbits / match_range);
-      auto mid = const_free_map_.lower_bound(const_nbits);
       auto end = const_free_map_.upper_bound(const_nbits * match_range);
-      for (auto it = mid; it != end; ++it) {
+      for (auto it = begin; it != end; ++it) {
```

Comment:
The mid logic is intentional, which allows us to start looking at the buffer that is bigger than the required size first, before start looking at smaller ones. A comment about this logic would be helpful

Reply:
ok